docs/_sections/buttons.html

```diff
@@ -76,7 +76,92 @@ <h3 id="linkButton">Link Button</h3>
 &lt;button class=&quot;nypl-link-button&quot;&gt;Cancel&lt;/button&gt;
 &lt;button class=&quot;nypl-basic-button&quot;&gt;Submit&lt;/button&gt;
 </code></pre>
+<!-- ////// -->
+<h3 id="shortButton">Short Buttons</h3>
+<p>This pattern is for use where the design intent requires a button that is shorter in height than the normal buttons available. In this example we use the <i>Short Button</i> pattern to activate a filter modal.</p>
+<div class="nypl-toolkit-example">
+  <span class="nypl-example-text">Example</span>
+  <button class="nypl-short-button" type="button" name="button">
+    Filter Results
+    <svg aria-hidden="true" class="nypl-icon" preserveAspectRatio="xMidYMid meet" viewBox="0 0 19 22">
+      <title>filter.icon.3</title>
+      <g>
+        <circle cx="6.65947" cy="2.31986" r="1.31924"/>
+        <circle cx="13.18733" cy="1.31986" r="1.31895"/>
+        <circle cx="9.56477" cy="5.46901" r="1.31927"/>
+        <g>
+          <path d="M7.74355,22.50683a.95047.95047,0,0,1-.95022-.95022V11.28645L.25259,4.23413A.95041.95041,0,1,1,1.64824,2.94366l7.04554,7.598v11.015A.95047.95047,0,0,1,7.74355,22.50683Z"/>
+          <path d="M11.60384,19.73881a.95047.95047,0,0,1-.95022-.95022V10.5478l7.126-7.81485a.95047.95047,0,0,1,1.41049,1.27439l-6.636,7.27293v7.50832A.95047.95047,0,0,1,11.60384,19.73881Z"/>
+        </g>
+      </g>
+    </svg>
+  </button>
+  <pre><code class="scss">.nypl-short-button
+ .nypl-icon</code></pre>
+  <pre><code class="html">
+&lt;button class=&quot;nypl-short-button&quot; type=&quot;button&quot; name=&quot;button&quot;&gt;
+  Filter Results
+  &lt;svg aria-hidden=&quot;true&quot; class=&quot;nypl-icon&quot; preserveAspectRatio=&quot;xMidYMid meet&quot; viewBox=&quot;0 0 19 22&quot;&gt;
+    &lt;title&gt;filter.icon.3&lt;/title&gt;
+    &lt;g&gt;
+      &lt;circle cx=&quot;6.65947&quot; cy=&quot;2.31986&quot; r=&quot;1.31924&quot;/&gt;
+      &lt;circle cx=&quot;13.18733&quot; cy=&quot;1.31986&quot; r=&quot;1.31895&quot;/&gt;
+      &lt;circle cx=&quot;9.56477&quot; cy=&quot;5.46901&quot; r=&quot;1.31927&quot;/&gt;
+      &lt;g&gt;
+        &lt;path d=&quot;M7.74355,22.50683a.95047.95047,0,0,1-.95022-.95022V11.28645L.25259,4.23413A.95041.95041,0,1,1,1.64824,2.94366l7.04554,7.598v11.015A.95047.95047,0,0,1,7.74355,22.50683Z&quot;/&gt;
+        &lt;path d=&quot;M11.60384,19.73881a.95047.95047,0,0,1-.95022-.95022V10.5478l7.126-7.81485a.95047.95047,0,0,1,1.41049,1.27439l-6.636,7.27293v7.50832A.95047.95047,0,0,1,11.60384,19.73881Z&quot;/&gt;
+      &lt;/g&gt;
+    &lt;/g&gt;
+  &lt;/svg&gt;
+&lt;/button&gt;
+  </code></pre>
+</div>
+<!-- ////// -->
+<h3 id="cancelFilters">Cancel Filter Button</h3>
+  <p>In a scenario where we have filters activated, we need to allow the user to unset each filter. The "x" icon is added inline and uses the <code>nypl-icon</code> class. </p>
+  <div class="nypl-toolkit-example">
+    <span class="nypl-example-text">Example</span>
+      <button class="nypl-unset-filter">Filter
```

Comment:
Maybe this should say 'cancel' to go along with the example.

Reply:
`nypl-cancel-filter`?

Reply:
Oh nevermind. I thought that "Filter" was the label but it's a placeholder for the name of any filter. All's good. 👍

Reply:
This is the design pattern...
![screen shot 2017-09-13 at 4 17 32 pm](https://user-images.githubusercontent.com/20174701/30398643-26ca3b9e-989f-11e7-8dd0-c0e92a721f50.png)

Reply:
regarding the checkbox style. I have a fix in on `checkbox-style` branch...

Reply:
I'd say fix the conflicts for this PR and merge it and then update.